bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/visitors/IdentifyLatestRecordRowNumVisitor.java

```diff
@@ -0,0 +1,112 @@
+package com.google.zetasql.toolkit.antipattern.parser.visitors;
+
+import autovalue.shaded.kotlin.Pair;
```

Comment:
Careful with using this Kotlin class. Java does not have a native `Pair<T, U>`, but we can just create a simple data class if we need that abstraction.